requirements.txt

```diff
@@ -0,0 +1,11 @@
+# List of external libraries required to run the scripts:
+
+colorama==0.4.6
```

Comment:
a po co to w requirementsach? 
Widze ze importujemy 
 import os
from dotenv import load_dotenv
import tkinter as tk

z czego czesc z nich juz mamy z bomby na systemie, wiec nie trzeba ich importowac wszystkich. Do zmiany

Reply:
done!